app/assets/stylesheets/style.scss

```diff
@@ -81,3 +81,24 @@ body {
   position: fixed;
   width: 60px;
 }
+
+.post-title {
+  font-size: 125%;
+  font-weight: bold;
+}
+
+.post-body {
+  font-size: 90%;
+  font-weight: normal;
+  margin: 10px;
+}
+
+.course-buttons
+{
+    width:100%;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered text-align, width<br>Colon after property should be followed by one space